### PR TITLE
Isambard2

### PR DESCRIPTION
**Pull request by heaton-dev**

### CODE REVIEW

**Comment by heaton-dev** [comment]
Renamed services, defluffed, add new nodes, updated values (core count, storage, etc), added placeholders, updated ssh config, added requirements.txt, cleaned up 🧹

File: index.rst [hunk 1]
```diff
@@ -9,10 +9,11 @@ Isambard User Documentation
 Isambard is a HPC service provided by `GW4 <http://gw4.ac.uk>`_ and the `UK Met Office <https://www.metoffice.gov.uk>`_.
 The system is funded by `EPSRC <http://www.epsrc.ac.uk>`_ and is one of six Tier-2 HPC facilities in the UK.
 
-Isambard is a Cray XC50 system which comprises 10,496 cores, and is one of the worlds first production Arm-based supercomputers.
-Each of the 164 compute nodes contains two 32-core Cavium ThunderX2 processors running at 2.1 GHz, and has 256 GB of DDR4-2666 memory.
-The nodes are connected via Cray Aries interconnect with a Dragonfly topology.
-A Cray Sonexion 3000 storage cabinet provides 480 terabytes of Lustre storage.
+Isambard is a Cray XC50 system which comprises 20'992 cores, and is one of the worlds first production Arm-based supercomputers.
+
+Each of the 329 compute nodes contain two 32-core Cavium ThunderX2 processors running at 2.1 GHz and have 256 GB of DDR4-2666 memory.
```
Comment (christopheredsall) [inline index.rst]: Cavium -> Marvell ?

Comment (christopheredsall) [inline index.rst]: There's now a mix of node memory sizes 256 GiB and 512 GiB:

```
br-cedsall@xcil00:~/aarch64/RSE/gw4-isambard/docs/pr-57> pbsnodes -a | grep resources_available.mem | sort | uniq -c
      1      resources_available.mem = 263986852kb
      1      resources_available.mem = 263986916kb
    160      resources_available.mem = 268435456kb
    169      resources_available.mem = 536870912kb
```

**Review by christopheredsall — APPROVED** [review]

File: user-guide/MACS.rst [hunk 2]
```diff
@@ -1,26 +1,41 @@
...
-* 2x Login nodes with Intel Xeon CPU
-* 4x GPU nodes with 2x Nvidia P100 "Pascal" GPUs and Intel Xeon E5-2695 v4 "Broadwell" CPU
-* 8x Intel Xeon Phi with "Knights Landing" 7210 CPU
-* 2x IBM Power 9 nodes
-* * Including 2x Nvidia V100 "Volta" GPU with NVLink
+* 2x Login nodes with Intel Xeon "Broadwell" CPU
+* 4x Nvidia Pascal GPU nodes with 2x Nvidia P100 "Pascal" GPUs and Intel Xeon E5-2695 v4 "Broadwell" CPU
+* 4x Nvidia Volta GPU nodes with 1x Nvidia V100 "Volta" GPU and Intel Xeon Gold 6230 "Cascade Lake (CXL)" CPU
+* 4x AMD Epyc 7742 "Rome" CPUs
+* 4x Intel Xeon 6230 "Cascade Lake (CXL)" CPU
```
Comment (tomgreen66) [inline user-guide/MACS.rst]: Should that be CLX?

File: user-guide/XCI.rst [hunk 3]
```diff
@@ -1,7 +1,7 @@
-Phase 2 - XC50 ARM
-------------------
+XCI - Marvell Thunder X2
+------------------------
 
-Cray XC50 cabinet, codenamed XCI, containing 10'000 ARM cores, available since November 2018.
+Cray XC50 cabinet, codenamed XCI, containing 20’992 ARM cores, available since November 2018.
```
Comment (tomgreen66) [inline user-guide/XCI.rst]: Should this say something like "initially installed November 2018, expanded September 2020?

File: user-guide/connecting.rst [hunk 4]
```diff
@@ -32,21 +36,37 @@ The following stanza is required in your local ``~/.ssh/config`` in order to tra
...
+ # Isambard: A64FX
+ Host gw4a64fxlogin00.isambard.gw4.ac.uk a64fxlogin00.isambard a64fx.isambard
+   Hostname gw4a64fxlogin00
+   ProxyCommand ssh isambard.gw4.ac.uk 'nc %h %p'
+
+ Host gw4a64fxlogin01.isambard.gw4.ac.uk a64fxlogin01.isambard
+   Hostname gw4a64fxlogin01
+   ProxyCommand ssh isambard.gw4.ac.uk 'nc %h %p'
+
+
```
Comment (tomgreen66) [inline user-guide/connecting.rst:48]: Will this confuse people mentioning this when we dont describe the system in its own section?

File: user-guide/jobs.rst [hunk 5]
```diff
@@ -15,13 +13,16 @@ Users can submit any number of jobs but only two jobs per-user per-queue will ru
...
 ===================
 
-* arm     - Run on 164x Thunder X2 XC50 compute nodes
-* arm-dev - Run interactively on up to 4x Thunder X2 XC50 compute nodes
-* knlq    - Run on 8x Intel Xeon Phi "Knights Landing" 7210 CPU nodes
-* pascalq - Run on 4x dual-card Nvidia Tesla P100 "Pascal" GPU nodes
-* powerq  - Run on 2x IBM Power 9 nodes, each with dual-card Nvidia V100 "Volta" GPUs ← ``Queue unavailable, interactive use only, hosts: power-001, power-002``
+* `arm`     - Run on XCI Marvell Thunder X2 nodes
+* `arm-dev` - Run interactively on up to 4x XCI Thunder X2 nodes
+* `rome`    - Run on MACS 4x AMD Epyc "Rome" 7742 nodes
```
Comment (tomgreen66) [inline user-guide/jobs.rst]: Think that should be romeq

File: user-guide/support.rst [hunk 6]
```diff
@@ -1,5 +1,5 @@
-Getting Support
-===============
+Support
+=======
 
 The first point of contact for all issues and queries is the Isambard Support helpdesk, which can be reached via email at isambard-support@gw4.ac.uk.
```
Comment (tomgreen66) [inline user-guide/support.rst:4]: Any problem making the email a mailto link?

File: user-guide/MACS.rst [hunk 7]
```diff
@@ -1,26 +1,41 @@
...
+Nvidia GPU
+==========
 
-`Intel Xeon Phi "Knights Landing" 7210 CPU <https://ark.intel.com/products/94033/Intel-Xeon-Phi-Processor-7210-16GB-1_30-GHz-64-core>`_ @ 1.30GHz (1.50GHz Turbo) with AVX-512.
+Isambard hosts two generations of Nvidia GPUs: `P100 "Pascal" <https://www.nvidia.com/en-us/data-center/pascal-gpu-architecture/>`_ & `V100 "Volta" <https://www.nvidia.com/en-us/data-center/volta-gpu-architecture/>`_.
+
+AMD Epyc
+========
+
+`AMD Epyc 7742 "Rome" <https://www.amd.com/en/products/cpu/amd-epyc-7742>`_ 2.25GHz 64-core CPU nodes with DDR4-3200Mhz over 8 memory channels.
```
Comment (tomgreen66) [inline user-guide/MACS.rst:28]: Should be mention threading and multi-socket nature of the nodes?

File: user-guide/MACS.rst [hunk 8]
```diff
@@ -1,26 +1,41 @@
...
+
+Intel Xeon "Cascade Lake"
+=========================
+
+`Intel Xeon Gold 6230 "Cascade Lake (CXL)" <https://ark.intel.com/content/www/us/en/ark/products/192437/intel-xeon-gold-6230-processor-27-5m-cache-2-10-ghz.html>`_ 2.10GHz 20-core CPU nodes with DDR4-2933MHz over 6 memory channels.
```
Comment (tomgreen66) [inline user-guide/MACS.rst]: Similar to above, think it has hyperthreading disabled but has 2 sockets.

File: user-guide/MACS.rst (hunk 2 above)
Comment (heaton-dev) [reply user-guide/MACS.rst]: Oops, correct, CXL is something else!

File: user-guide/XCI.rst (hunk 3 above)
Comment (heaton-dev) [reply user-guide/XCI.rst]: I feel like the docs should represent the current state, leave the history to a blog post

File: user-guide/support.rst (hunk 6 above)
Comment (heaton-dev) [reply user-guide/support.rst:4]: Sphinx does this automagically

File: user-guide/MACS.rst (hunk 7 above)
Comment (heaton-dev) [reply user-guide/MACS.rst:28]: I'll leave that to someone else 😉

File: user-guide/MACS.rst (hunk 8 above)
Comment (heaton-dev) [reply user-guide/MACS.rst]: I'll leave that to someone else 😉

**Pull request merged**